app/assets/javascripts/float-kit/addon/components/d-default-toast.gjs

```diff
@@ -0,0 +1,61 @@
+import Component from "@glimmer/component";
+import DButton from "discourse/components/d-button";
+import icon from "discourse-common/helpers/d-icon";
+import { concat, fn } from "@ember/helper";
+import concatClass from "discourse/helpers/concat-class";
+import or from "truth-helpers/helpers/or";
+import { action } from "@ember/object";
+
+export default class DDefaultToast extends Component {
+  <template>
+    <div
+      class={{concatClass
+        "d-default-toast"
+        (concat "-" (or @model.theme "default"))
+      }}
+      ...attributes
+    >
+      {{#if @model.icon}}
+        <div class="d-default-toast__icon-container">
+          {{icon @model.icon}}
+        </div>
+      {{/if}}
+      <div class="d-default-toast__main-container">
+        <div class="d-default-toast__texts">
+          {{#if @model.title}}
+            <div class="d-default-toast__title">
+              {{@model.title}}
+            </div>
+          {{/if}}
+          {{#if @model.message}}
+            <div class="d-default-toast__message">
+              {{@model.message}}
+            </div>
+          {{/if}}
+        </div>
+
+        {{#if @model.actions}}
+          <div class="d-default-toast__actions">
+            {{#each @model.actions as |toastAction|}}
+              <DButton
+                @icon={{toastAction.icon}}
+                @translatedLabel={{toastAction.label}}
```

Comment:
Is there a reason you went with `translatedLabel` here instead of just `label` and passing a string?

Reply:
it can come from the outside:

`<DToast @options={{hash label="xxx"}} />`

Reply:
because you don't know? if `toastAction.label` is already translated or not

Reply:
Sure, I am just not sure we need to care

eg

https://github.com/discourse/discourse/blob/9db047a76cbb554469ea23351a04db31ac271662/app/assets/javascripts/discourse/app/components/d-modal.hbs#L51

we don't check on d-modal if you have passed a translated string or not

Reply:
Great 👍